### PR TITLE
Add debug.setmemorycategory

### PR DESCRIPTION
**Pull request by ffrostflame**
I am not sure if I did this right. It's my first time

### CODE REVIEW

**Review by Kampfkarren — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## Unreleased
 
+## [0.18.0] - 2022-04-19
+
+## Added
+- Added `debug.setmemorycategory` and `debug.setmemorycategory`
```
[the remark this answers is not in the document]
Comment (ffrostflame) [reply CHANGELOG.md:8]: alright

**Comment by Kampfkarren** [comment]
It doesn't look like you did either of my changes?

**Comment by Kampfkarren** [comment]
You have now broken the original 0.18.0 changelog :P

**Comment by Kampfkarren** [comment]
I misspoke, but both of your commits are still wrong.

There is NO 0.18.0 changelog, remove it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+## Added
```
Comment (Kampfkarren) [inline CHANGELOG.md:5]:
```suggestion
### Added
```

**Comment by Kampfkarren** [comment]
Should be good now, will merge when CI passes.

**Pull request merged**